LoadEnvVariables/LoadEnvVariables.cs

```diff
@@ -0,0 +1,48 @@
+﻿using DotNetEnv;
+using Newtonsoft.Json.Linq;
+
+namespace LoadEnvVariables;
+
+public class AzureEnvManager
+{
+    private const string ConfigFilePath = "../../.azure/config.json";
```

Comment:
I think we can handle the paths better.  Like `../../.azure` is in a var and then we build off of that for the config.json and the .env